stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -202,5 +202,8 @@ public interface ChatDomain {
         @JvmStatic
         public fun instance(): ChatDomain = instance
             ?: throw IllegalStateException("ChatDomain.Builder::build() must be called before obtaining ChatDomain instance")
+
+        public val isInitialized: Boolean
```

Comment:
What's about to make it JvmStatic?

Reply:
We have a `isInitialized` instance method already in `ChatDomain` that it would clash with. Plus I don't particularly want anyone to use this API, so it's not worth making friendly for Java clients 😅